object_tracker/object_tracker.py

```diff
@@ -77,12 +88,45 @@ def run_frame(self, img: ndarray) -> Tuple[ndarray, List[int], str, float]:
             tracker_output = self._tracker.run_frame(orig)
             if tracker_output is None:
                 return img, None
-
+            
             min_x, min_y, max_x, max_y, flag, score = tracker_output
+
+
+
+            if self.run_of_low_score:
+                if self._tracker_counter % self._tracker_run_iter != 0:
+                    try:
+                        optical_flow_output = self._match(orig)
+                    except Exception as e:
+                        logging.error(e)
+                        logging.error("match raise exception")
+                    if optical_flow_output is not None:
+                        if score < self.score_thresh:
```

Comment:
The optical flow also finds new feature, if we would run OF only on low score it won't find any features to the last frame because the scene would change too much.